bindings/python/Makefile

```diff
@@ -45,31 +44,46 @@ sdist3:
 	rm -rf src/ dist/
 	rm -rf prebuilt/win64/unicorn.dll
 	rm -rf prebuilt/win32/unicorn.dll
-	cp README.pypi-src README
 	cp PKG-INFO.src PKG-INFO
 	python3 setup.py sdist register upload
 
+# build & upload PyPi package with precompiled core
+bdist:
+	rm -rf src/ dist/
+	rm -rf prebuilt/win64/unicorn.dll
+	rm -rf prebuilt/win32/unicorn.dll
+	cp PKG-INFO.src PKG-INFO
```

Comment:
This wasn't copied from capstone, this was copid from the `sdist` code six lines up :P